test/e2e/beat/webhook_test.go

```diff
@@ -0,0 +1,36 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package beat
+
+import (
+	"testing"
+
+	beatv1beta1 "github.com/elastic/cloud-on-k8s/pkg/apis/beat/v1beta1"
+	"github.com/elastic/cloud-on-k8s/test/e2e/test"
+	"github.com/stretchr/testify/require"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func TestWebhook(t *testing.T) {
+	beat := beatv1beta1.Beat{
```

Comment:
Would it make sense to follow the established pattern here: https://github.com/elastic/cloud-on-k8s/blob/ccaa19f04956e52ec543f46806640822c4440e57/pkg/apis/enterprisesearch/v1beta1/webhook_test.go#L21

Reply:
I realise after talking to @david-kow that this is not actually testing correct deployment of the webhook. So this can be a different PR.